open-company/open-company-web
# Org name max length

Review with:
- [Storage](https://github.com/open-company/open-company-storage/pull/205)

To test:
- change `oc.web.utils.org:8` to be 60 instead of 50
- start web
- to go signup
- enter email and password
- when you get to `Tell us about you` step
- inspect the org name field
- change the max-length prop of the DOM input element to be 100
- now insert your first and last name
- insert a company name of 100 chars
- [x] can't do that? name is always cut at 60? Good
- now create the team
- [x] did you get an error about the org name? Good
- enter a company name of 50 chars
- finish onboard
- go to settings
- change org name
- inspect the org name field and change the `max-length` property to be 100
- [x] can NOT change with an org name less than 3 chars?
- [x] can NOT change with an org name more than 50 chars?
- change again `oc.web.utils.org:8` to 50
- logout
- signup again
- [x] make sure the UI doesn't let you add a company name longer than 50 chars
- finish onboard
- go to settings
- [x] make sure the UI doesn't let you change the org name with something longer than 50 chars


Additional tests for routing library update (Secretary):
- go to /login (logged out, this ensure you are suing the React version of the app, not the static)
- dismiss the login popover
- nav to about from the top menu
- nav to pricing from the top menu
- signup via email
- signup via Google
- signup via Slack
- login via email
- login via Google
- login via Slack
- invite an email user and signup with that user
- reset password
- opening `/org/all-posts?org-settings=org` show the org settings panel?
- opening `/org/all-posts?user-settings=profile` show the user settings panel?
- opening `/org/all-posts?reminders=1` show the reminders panel?
- visit AP directly (no internal nav)
- visit a board directly (no internal nav)
- switch between boards, Drafts and AP
- share a post and copy the secure URL
- open it in incognito mode
- get a digest url (with id) and open it
- open a route that should 404
